website/components/PropTable.tsx

```diff
@@ -90,7 +83,7 @@ function PropTable({
   markdownAst,
   component,
 }: {
-  markdownAst: Node;
+  markdownAst: CellRoot;
```

Comment:
`CellRoot` represents a table cell. It doesn't make much sense to reuse it here. The root of a readme file is a unique type `ReadmeMarkdown` whose structure can be hinted at by how it's used

Reply:
👍